docs/user/sbt.rst

```diff
@@ -12,21 +12,21 @@ The easiest way to make a fresh project is to use our official gitter8 template:
 
     sbt new scala-native/scala-native.g8
 
-This includes:
+This includes the following files with the <version> filled in:
 
 * ``project/plugins.sbt`` to add a plugin dependency::
 
-    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.7")
+    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "<version>")
 
 * ``project/build.properties`` to specify the sbt version::
 
-    sbt.version = 0.13.17
+    sbt.version = <version>
```

Comment:
This is seems wrong. Sbt version can not the same as Scala Native version.

Reply:
Yes, agreed. See if the change makes more sense. I wanted to remove the versions that could easily get out of date.